src/utils/debugging-and-logging.ts

```diff
@@ -23,91 +18,24 @@ export class M {
         return moment().format("MM.DD.YY HH:mm:ss");
     }
     static log(...args: any[]) {
-        process.stdout.write(`[${M.get_timestamp()}] [log]   `);
+        process.stdout.write(`[${M.get_timestamp()}] [log]     `);
         console.log(...args, `(from: ${get_caller_location()})`);
     }
     static debug(...args: any[]) {
-        process.stdout.write(`${chalk.gray(`[${M.get_timestamp()}] [debug]`)} `);
+        process.stdout.write(`${chalk.gray(`[${M.get_timestamp()}] [debug] 🐞`)} `);
```

Comment:
I'm a tiny bit worried about how these will look in the terminal / while browsing log files, but happy to merge and see how it looks

Reply:
I switched the 🐞 out for 🛠️, moved them to the start of the line, and fixed the alignment. It looks a lot better now. I don't think there should be an issue with log files, it's just UTF-8 in the end anyways?